src/scheduler/src/test/java/org/cloudfoundry/autoscaler/scheduler/rest/ScheduleRestControllerCreateScheduleAndNofifyScalingEngineTest.java

```diff
@@ -133,6 +133,8 @@ public void testDeleteSchedule() throws Exception {
         mockMvc.perform(
             delete(TestDataSetupHelper.getSchedulerPath(appId)).accept(MediaType.APPLICATION_JSON));
 
+    java.util.concurrent.TimeUnit.SECONDS.sleep(5); // ⏳ Waiting time to ensure, the db had …
+    // … enough time to delete the job.
```

Comment:
Actually I wanted to write rather
```suggestion

    // ⏳ Waiting time to ensure, the db had enough time to delete the job.
    java.util.concurrent.TimeUnit.SECONDS.sleep(5);

```
or at least
```suggestion
    java.util.concurrent.TimeUnit.SECONDS.sleep(5); // ⏳ Waiting time to ensure, the db had …
                                                    // … enough time to delete the job.
```
But that java-style-guide does not like it.